medicines/doc-index-updater/src/pars_upload.rs

```diff
@@ -10,8 +11,8 @@ use serde::{Deserialize, Serialize};
 use bytes::BufMut;
 use futures::future::join_all;
 use futures::TryStreamExt;
-use jsonwebtoken::{dangerous_unsafe_decode, errors::Error};
-use search_client::models::DocumentType;
+use jsonwebtoken::dangerous_unsafe_decode;
+use search_client::{get_env, models::DocumentType};
```

Comment:
seems a weird place to get `get_env` from but I guess we were already doing this sort of thing